source/unified-test-format/unified-test-format.rst

```diff
@@ -1743,14 +1771,19 @@ to block indefinitely. This should not be a concern for
 ``iterateUntilDocumentOrError`` as iteration only continues until either a
 document or error is encountered.
 
-Test runners MUST ensure that this operation will not inadvertently skip the
-first document in a cursor. Albeit rare, this could happen if
-``iterateUntilDocumentOrError`` were to blindly invoke ``next`` (or
-equivalent) on a cursor in a driver where newly created cursors are already
-positioned at their first element and the cursor had a non-empty
-``firstBatch`` (i.e. ``resumeAfter`` or ``startAfter`` used). Alternatively,
-some drivers may use a different iterator method for advancing a cursor to
-its first position (e.g. ``rewind`` in PHP).
+iterateOnce
+~~~~~~~~~~~
+
+Performs a single iteration of the cursor. If the cursor's current batch is
+empty, one ``getMore`` MUST be attempted to get more results. This operation
+takes no arguments. If `expectResult <operation_expectResult_>`_ is
+specified, it SHOULD be a single document.
```

Comment:
For the benefit of others that might review this, I'll note that SHOULD is being used because the schema does not enforce this so the best we can do is ask folks not to write such tests.